README.md

```diff
@@ -83,6 +83,9 @@ table includes the currently implemented options.
 | tags | "smoketest,regression"  | See description above|
 | output | c:\\temp\\reports  | Directory to store reports in.|
 | reportFileName |  N/A | File name for the report. If omitted, the report will get a name based on the test job ID|
+| callback | http://localhost:8080 | The URL, to which the results will be posted|
+| environment | dev | Specify the The target environment for test job. Can not be used with **endpoint**|
+| endpoint | localhost:8080 | The endpoint to be used for HTTP requests sent by this test, in the format host:[port]. Can not be used with **environment**|
```

Comment:
**endpoint** should be called hostAndPort, the way it is in our API

Reply:
All occurrences of endpoint have been replaced with hostAndPort.